regression/scripts/rocket_network/Rocketdyne_J2.py

```diff
@@ -0,0 +1,205 @@
+# Rocketdyne_J2.py
```

Comment:
Wait do these regressions cover different things?

At first glance they look similar.

Reply:
They both regress the liquid rocket network, but with different propellant types. Im not sure if a propellant regression should be created (seems kinda dumb) or another rocket (overkill).  Ill go with what you think is best.